x-pack/plugins/cases/public/components/visualizations/actions/utils.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { IEmbeddable } from '@kbn/embeddable-plugin/public';
+import { LENS_EMBEDDABLE_TYPE, type Embeddable as LensEmbeddable } from '@kbn/lens-plugin/public';
+import type { ApplicationStart } from '@kbn/core/public';
+import { OWNER_INFO } from '../../../../common/constants';
+import { canUseCases } from '../../../client/helpers/can_use_cases';
+import type { DashboardVisualizationEmbeddable } from './types';
+import type { CasesPermissions } from '../../../../common';
+
+export const isLensEmbeddable = (embeddable: IEmbeddable): embeddable is LensEmbeddable => {
+  return embeddable.type === LENS_EMBEDDABLE_TYPE;
+};
+
+export const getCaseOwner = (currentAppId?: string) =>
+  Object.values(OWNER_INFO)
+    .filter((info) => info.appId === currentAppId)
```

Comment:
I don't think it should be possible for a `currentAppId` to match multiple `info.appId` entries. Could we use a find instead so we just return a single owner instead of an array?